### DISCUSSION TITLE
TIOBE Index

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#102] [General]
The TIOBE index analyzes the demand for specific software languages (20 years in a row). Here is a link to the [July 2021 report](https://www.tiobe.com/tiobe-index/)

C is at the top followed by Java and Python. Clojure is in the +50 category which is unranked. 

Avi and I were talking about learning Ruby and how "popular certain languages are over other languages. Does this have any impact on what language we decide to code in? Probably not, but it's interesting. What languages have you used and what are the benefits you've found between them?

**Comment by avidrucker** [comment]
I definitely think our perceptions, attitudes, and judgements, individually and collectively, have an impact on our decisions. I decided to learn Python initially because that seemed like a friendly beginner language, and, for the most part, it was... until it wasn't. I "language hopped" for a number of years until I started working in JavaScript and developed professional competency in it. My current favorite language is Clojure because I see it as a more mature choice, and long-term effective tool for me, my career, and my positive impact in the world.
